plotter.go

```diff
@@ -42,7 +42,7 @@ func (p SimplePlotter) Plot(cc clusters.Clusters, iteration int) error {
 	for i, c := range cc {
 		series = append(series, chart.ContinuousSeries{
 			Style: chart.Style{
-				Show:        true,
+				Hidden:		 false,
```

Comment:
Is this an upstream API change? Maybe we can just omit setting `Hidden` here and below, as `false` should already be its default value.

Reply:
I Also though of that, but let it in just to be sure it isn't initiated with something weird.

But sure we can also just omit it as well, wouldn't harm i guess.